controllers/wavefrontalert_controller.go

```diff
@@ -105,39 +133,123 @@ func (r *WavefrontAlertReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		if exist && &wfAlert.Status.ExportParamsChecksum != nil {
 			if reqChecksum != wfAlert.Status.ExportParamsChecksum {
 				proceed = false
+				status.State = alertmanagerv1alpha1.ReadyToBeUsed
```

Comment:
We probably need a zoom meeting to go through this ExportParams logic with some examples, let's talk next week and include Nathan as well